pkg/configv2/default.go

```diff
@@ -0,0 +1,213 @@
+package configv2
+
+import (
+	"runtime"
+	"time"
+
+	"github.com/bacalhau-project/bacalhau/pkg/configv2/types"
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+const (
+	Day    = types.Duration(time.Hour * 24)
+	Hour   = types.Duration(time.Hour)
+	Minuet = types.Duration(time.Minute)
+	Second = types.Duration(time.Second)
+)
+
+// NB(forrest): setting copied verbatim from:
+// https://www.notion.so/expanso/Rethinking-Configuration-435fbe87419148b4bbc5119d413786eb?pvs=4#6a28290e0c514e3b95e8ec6ee0106379
+
+// Default is the default configuration for a bacalhau node.
+var Default = types.Bacalhau{
+	API: types.API{
+		Address: "0.0.0.0:1234",
+	},
+	DataDir:            "~/.bacalhau",
+	NameProvider:       "puuid",
+	StrictVersionMatch: false,
+	Orchestrator: types.Orchestrator{
+		Enabled: true,
+		Listen:  "0.0.0.0:4222",
+		Cluster: types.Cluster{
+			Listen: "0.0.0.0:4222",
+		},
```

Comment:
clustering should be disabled by default. This enables orchestrators to connect to each other which we are not fully supporting yet

Reply:
It is used when spinning up requester node, and nodes do connect with each other, but we don't really support multiple requesters where they share some resonasbles or do leader election just yet